src/components/adaptive-cards/InputToggle/InputToggle.jsx

```diff
@@ -8,16 +10,60 @@ import {acPropTypes, registerComponent} from '../Component/Component';
  *
  * @param {object} props  React props passed to the component
  * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Inline style for the component
  * @returns {object} JSX of the component
  */
-export default function InputToggle({data}) {
+export default function InputToggle({data, className, style}) {
+  const [cssClasses, sc] = webexComponentClasses('adaptive-cards-input-toggle', className);
+  const {getValue, setInput} = useContext(AdaptiveCardContext);
+  const inputValue = getValue() || 'false';
+  const [checked, setChecked] = useState(Boolean(data.value) || false);
```

Comment:
no state here, use setValue from context, otherwise the validation and submit may work incorrectly